### ISSUE TITLE
KhuzaitCavalryAgility wrong calculation

### ISSUE THREAD

**Issue by peppermor** [Tyler-IN/MnB2-Bannerlord-CommunityPatch#224]
Hi, as was planing to play as Horse Lord I tested your fix for this feat and what i can see is that it calculate wrongly. In tooltip "Party Speed" I have 4.99+2.99+0.38=5 :) Issue only exist palying as Khuzait with other mods and with CommunityPAtch only. Tested both on existing save and on new game same result. Thanks for fixing all of this perks and feat, you guys are great!

**Comment by Tyler-IN** [comment]
Could you post the version of Community Patch you were using?

**Comment by peppermor** [comment]
I builded it fromlatest master and latest dev branches same situation on both.

**Comment by iPherian** [comment]
Can you see if the problem occurs without mod enabled?

**Comment by peppermor** [comment]
BTW do you have any schedule for release? Maybe more frequent smaller changes?

**Comment by iPherian** [comment]
So just to be clear, it doesn't happen when you have all mods disabled?

**Comment by peppermor** [comment]
It happens when I use all of my mods and when I use only CommunityPatch. Without any mods cultural feat do not work so party speed is calculated like for any other party/culture.

**Comment by peppermor** [comment]
It looks like it do not take bonus from cavalry amount in party when calculate final value.

**Comment by fnzr** [comment]
If you buy a horse, does the tooltip updates to the correct value?

**Comment by peppermor** [comment]
> 
> 
> If you buy a horse, does the tooltip updates to the correct calculations?

No
I've done some testing and I can confirm that for Khuzaits only "Base" and  "KhuzaitCavalryAgility" is taken. 
Adding horses, removing horses, adding infrantry, going on foot with main char give same result. If i am solo on horse my speed:
Base: 4.99
Cavalry: +2.99
Game difficulty: +0.5
Cargo: -0.02
KhuzaitCavalryAgility: +0.3
Snow: -0.01
Total: +5.28

Solo on foot:
Base: 4.99
Game difficulty: +0.5
Cargo: -0.02
Snow: -0.01
Total +4.98

**Comment by peppermor** [comment]
Tested it also for Battania bonus and everything adds up even to much because here forest penalty is not substracted so you can gallop 8.63 in the middle of woods. This bonus is added correctly all of rest componets also only forest penalty was skipped.

**Comment by fnzr** [comment]
yeah, I can reproduce this. The current patch only takes the base speed of the party and khuzait agility bonus in consideration:
```csharp
 var explainedNumber = new ExplainedNumber(baseNumber, explanation, null);
explainedNumber.AddFactor(khuzaitRatioModifier, efaultFeats.Cultural.KhuzaitCavalryAgility.Name);
__result = explainedNumber.ResultNumber;
```
At the very least, `baseNumber` should be `__result`, but I think there will be a few details that still don't add up.

**Comment by peppermor** [comment]
It looks like TW fixed feats in 1.3.0 now?

**Comment by peppermor** [comment]
Fixed in 0.1.0-rc1
